chat/src/main/java/bisq/chat/notifications/ChatNotificationService.java

```diff
@@ -245,7 +246,8 @@ private <M extends ChatMessage> void chatNotificationAdded(ChatChannel<M> chatCh
         }
 
         // Only notify for the currently selected channel
-        if (!chatService.getChatChannelSelectionService(chatChannel.getChatChannelDomain()).getSelectedChannel().get().equals(chatChannel)) {
+        if (chatChannel instanceof PublicChatChannel &&
```

Comment:
With PublicChatChannel we apply it the the common channels as well. I guess there its ok to show all notifications. Users can disable them anyway per channel. If applied only for BisqEasyOfferbookChannel the instanceof check should be using BisqEasyOfferbookChannel. I think only applying the filtering to BisqEasyOfferbookChannel is better.

Reply:
Implemented as suggested.

I think it would be better to just show number of unread messages in non selected channels, but as you say, it's possible to disable per channel. We could set the default for public channels to not show notifications, unless the intention is to get users engaged with the app and feel there is something happening. Maybe something to consider for a later release when notifications start being a nuisance rather than engaging.

Reply:
I just made a PR which changed the red color to grey for the default notification, so they do not get that annoying. 
I think in the beginning its better to boost the other common channels so ppl use it. As you said if it gets too much over time we can change default setting to turn them off. Also there will be further refinement for the whole notification design, but atm I think its better to focus on the most important...